code/modules/mob/living/simple_animal/slime/slime_atmos.dm

```diff
@@ -109,13 +109,21 @@
 		clear_friends()
 		absorb_gas(air, /datum/gas/tritium, 5)
 /**
- * BZ-exposed slimes gain a temporary status effect that makes them pacifist, docile, and be friends with everyone they see.
- * They also slowly lose charge over time.
+ * BZ-exposed slimes fall into stasis!
  */
 /mob/living/simple_animal/slime/proc/bz_expose(current_gas_pp, datum/gas_mixture/air)
+	if(current_gas_pp > 0.05 && (bodytemperature > (T0C + 100))) // if air is hot and bz is here, sleepy
```

Comment:
Prior to this pr the check was if the slime was _colder_ then 100c then above 100c , I believe so some trace BZ in the air during a plasma fire doesnt put slimes asleep when it should be their strongest moments.